### PR TITLE
Symphony Testnet: update asset list to include testnet's test stablecoins

### PR DESCRIPTION
**Pull request by dblackstone1**
Add USD, HKD, and VND basics to test IBC channels, swaps, and wallet integrations

### CODE REVIEW

File: testnets/symphonytestnet/assetlist.json [hunk 1]
```diff
@@ -31,6 +29,90 @@
...
+          "aliases": ["harmony USD"]
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "uusd",
+      "name": "Harmony USD",
+      "display": "husd",
+      "symbol": "HUSD",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/coin_icon.png"
```
Comment (JeremyParish69) [inline testnets/symphonytestnet/assetlist.json:53]: Please include these images

Comment (dblackstone1) [reply testnets/symphonytestnet/assetlist.json]: added the images

File: testnets/symphonytestnet/assetlist.json [hunk 2]
```diff
@@ -24,13 +22,97 @@
...
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/husd.png"
+        }
+      ]
+    },
+    {
+      "description": "Harmonized HKD",
+      "denom_units": [
+        {
+          "denom": "ukhd",
```
Comment (JeremyParish69) [inline testnets/symphonytestnet/assetlist.json:65]: everywhere ielse it seems to be HKD, but in the base and display denoms, it KHD. Just want to make sure there isn't a typo.

Comment (dblackstone1) [reply testnets/symphonytestnet/assetlist.json:65]: it is supposed to be hkd, yes.  unfortunately, the dev did have a typo and that is out on the current version of the testnet.  it'll be amended before mainnet and when this moves out of config into something a bit easier to manage dynamically

**Pull request merged**